docker-compose.yml

```diff
@@ -0,0 +1,103 @@
+# Use postgres/example user/password credentials

```

Comment:
It would be nice to modify the file, than deleting and creating a new one from a git history point of view. It will also be easy to compare what exactly changed.

Reply:
I'm not sure the GitHub mobile view here is leaving out something I'm not sure what the suggestion is.

Reply:
What I meant is, that we already have a docker-compose.yaml file. So it would be nice to modify the same file instead of deleting it and creating a new file with the same functionality. This will help people understand better what exactly was changed in a particular revision.